pkg/buildcontext/git.go

```diff
@@ -44,3 +54,11 @@ func (g *Git) UnpackTarFromBuildContext() (string, error) {
 	_, err := git.PlainClone(directory, false, &options)
 	return directory, err
 }
+
+func getGitPullMethod() string {
+	gitPullMethod := os.Getenv(gitPullMethodEnvKey)
+	if ok := supportedGitPullMethods[gitPullMethod]; !ok {
+		gitPullMethod = gitPullMethodHTTPS
```

Comment:
```suggestion
		return gitPullMethodHTTPS
```

Reply:
Thanks @tejal29  for your suggestion, you may commit that.